js/app-user.js

```diff
@@ -38,7 +38,7 @@ const Login = {
     // verify user's logged in status with Pulse API
     Service.userstatus()
       .then(response => {
-        setUserData(false, response.username, response.customname, response.email, response.moderator);
+        setUserData(false, response.username, response.profileid, response.customname, response.email, response.loggedin, response.moderator);
```

Comment:
There are way too many arguments here. We should probably just pass in `response` object here and fix other functions correspondingly (maybe change the `update` function parameters to `update(error, user)`)